Gems/GraphCanvas/Code/Source/Translation/TranslationSerializer.cpp

```diff
@@ -81,6 +81,18 @@ namespace GraphCanvas
                     {
                         itemKey.append(AZStd::string::format(".%d", i));
                     }
+
+                    rapidjson::Value::ConstMemberIterator innerContextItr = element.FindMember(Schema::Field::context);
+                    if (innerContextItr != element.MemberEnd())
+                    {
+                        itemKey.append(AZStd::string::format(".%s", innerContextItr->value.GetString()));
+                    }
+
+                    rapidjson::Value::ConstMemberIterator innerVariantItr = element.FindMember(Schema::Field::variant);
+                    if (innerVariantItr != element.MemberEnd())
+                    {
+                        itemKey.append(AZStd::string::format(".%s", innerVariantItr->value.GetString()));
+                    }
```

Comment:
Will this cause any backward breaking issue? as previously we didn't include context and variant as item key in db?

Reply:
No because none of the entries had them, if we identify any new nodes that are incorrect the process to fix them is trivial and we can do it on a need basis